google/cloud/storage/_media/requests/_request_helpers.py

```diff
@@ -23,8 +23,9 @@
 
 import time
 
-from google.resumable_media import common
-from google.resumable_media import _helpers
+from google.cloud.storage.exceptions import InvalidResponse
+from google.cloud.storage._media import common
+from google.cloud.storage._media import _helpers
```

Comment:
As discussed offline, I think we can keep them as-is, because the use of the two helpers modules is totally separated by being in _media or not already, so there are no points of conflict. However, it's a good idea for the future when the two sections of the library become more intertwined. As it's a private module, it's not necessary to front-load that work.